src/WebJobs.Extensions.DurableTask/Storage/NameValidator.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
```

Comment:
nit: I think we're missing a #nullable enable here.
```suggestion
#nullable enable
```